ArticleTemplates/assets/js/modules/experiments/lowFrictionParticipation.js

```diff
@@ -0,0 +1,263 @@
+        itemCount: 5,
+        templateVars: {
+            title: 'Do you agree? Rate this film now',
+            description: 'Let us know what you think!',
+            testMessage: 'Thanks for voting. This is a test. We\'re currently evaluating this as a potential new feature.'
+        }
+    };
+
+    var els = {};
+
+    var buttonPositions = [];
+
+    var touchStart;
+
+    var touchMove;
+
+    function init(options) {
+        if (isValidArticleType() && hasCommentsEnabled()) {
+            settings = GU.util.merge(settings, options);
+
+            els.lowFricContainer = document.createElement('div');
+            els.lowFricContainer.classList.add('participation-low-fric');
+            els.articleBody = document.querySelector('.article__body > .prose');
+
+            getUserVote(setUpParticipation);
+        }
+    }
+
+    function isValidArticleType() {
+        return GU.opts.contentType === 'article' && GU.opts.sectionTone === 'review' && GU.opts.section === 'film';
+    }
+
+    function hasCommentsEnabled() {
+        return document.querySelector('.comment-count');
+    }
+
+    function getUserVote(callback) {
+        window.retrieveLowFrictionParticipationData = callback;
+        GU.util.signalDevice('getTemplateStorage/' + storageKey + '/retrieveLowFrictionParticipationData');
+    }
+
+    function updateState(state) {
+        render(GU.util.merge(currentState, state));
+    }
+
+    function render(state) {
+        var reviewRating;
+
+        if (state.initialRender) {
+            els.lowFricContainer.innerHTML = 
+                '<h2 class="participation-low-fric__title">' + settings.templateVars.title + '</h2>' +
+                '<p class="participation-low-fric__desc">' + settings.templateVars.description + '</p>' +
+                '<div class="participation-low-friction__contents"><p class="review-rating"></p></div>';
+
+            reviewRating = els.lowFricContainer.querySelector('.review-rating');
+
+            reviewRating.innerHTML = createButtonsHTML();
+
+            els.articleBody.appendChild(els.lowFricContainer);
+        } else {
+            reviewRating = els.lowFricContainer.querySelector('.review-rating');
+            clearRating(reviewRating);
+        }
+
+        if (state.selectedItem) {
+            reviewRating.classList.add(state.selectedItem);
+        }
+    }
+
+    function createButtonsHTML() {
+        var i;
+        var html = '';
+
+        for (i = 0; i < settings.itemCount; i++) {
+            html += '<span data-icon="&#xe036;" aria-hidden="true" class="participation-low-fric--button"></span>';
+        }
+
+        return html;
+    }
+
+    function clearRating(reviewRating) {
+        var className;
+        var i;
+
+        for (i = reviewRating.classList.length; i > 0; i--) {
+            className = reviewRating.classList[i - 1];
+
+            if (className.indexOf('review-rating--') !== -1) {
+                reviewRating.classList.remove(className);
```

Comment:
Could you `break;` this here to avoid looping the remaining classes once you're done?